releases/v20.1.0-beta.4.md

```diff
@@ -0,0 +1,123 @@
+- Added new internal tables `crdb_internal.node_transactions` and `crdb_internal.cluster_transactions` that contain some metadata about active user transactions. [#46206][#46206]
+- Added the column `txn_id` to the `crdb_internal.node_queries` and   `crdb_internal.cluster_queries` tables. These fields represent the transaction ID of each query in each row. [#46206][#46206]
+- Columns in the process of being added or removed to a table are now always set to their default or computed value if another transaction concurrently [`INSERT`](../v20.1/insert.html)s, [`UPDATE`](../v20.1/update.html)s, or [`UPSERT`](../v20.1/upsert.html)s a row. This fixes an issue where a column being backfilled would not get properly set by concurrent transactions. [#46285][#46285]
+- Modified the `get_bits()` and `set_bit()` [builtin](../v20.1/functions-and-operators.html) functions to support [byte array](../v20.1/sql-constants.html#byte-array-literals). [#46380][#46380]
+- [`ROLLBACK TO SAVEPOINT`](../v20.1/rollback-transaction.html) (for either regular savepoints or "restart savepoints" defined with `cockroach_restart`) now causes a "feature not supported" error after a DDL statement in a HIGH PRIORITY transaction, in order to avoid a transaction deadlock. See issue [#46414][#46414] for details. [#46415][#46415]
+- CockroachDB now supports the `stddev_samp` aggregate [builtin](../v20.1/functions-and-operators.html) function, which is the same as `stddev` (according to PostgreSQL documentation, the latter is actually the historical alias of the former). [#46279][#46279]
+
+### Command-line changes
+
+- Ensured the correct error messages are shown to the user when using `nodelocal upload`. [#46311][#46311]
+
+### Bug fixes
+
+- Fixed a crash when [`IMPORT`](../v20.1/import.html)ing a table without a table definition. [#46193][#46193]
+- Queries with qualified stars that refer to tables in outer scopes now work. [#46233][#46233]
+- Fixed an incorrect "no data source matches prefix" error in some cases involving subqueries that use views. [#46226][#46226]
+- Previously, the `experimental_strftime` and `experimental_strptime` [builtin](../v20.1/functions-and-operators.html) functions used the non-POSIX standard %f for nanosecond display. However, as CockroachDB only supports up to microsecond precision and Python's `strftime` has %f to microsecond, we have similarly switched %f to use microsecond instead of nanosecond precision. [#46263][#46263]
+- Added a check that detects invalid sequence numbers in the RocksDB write-ahead log and returns an error during start instead of applying the invalid log entries. [#46328][#46328]
+- [Follower reads](../v20.1/follower-reads.html) that hit intents no longer have a chance of entering an infinite loop. This bug was present in earlier versions of the v20.1 release. [#46234][#46234]
```

Comment:
Suggest "fixed a bug where".

Also it makes me wonder if this bug was *only* in earlier 20.1 releases?  This wording could mean it's also present elsewhere.